### DISCUSSION TITLE
New to Knative

### DISCUSSION THREAD

**Discussion by sanzzz** [knative/community#1608] [General]
HI Everyone,
 We are planning to implement knative as it for serverless workloads and cost saving. Would like to understand few things here
Serverless mean - We dont manage servers instead cloud providers will take care of servers.  I believe pods are created when there is  a request and serve the traffic and scale down to zero - this is handled by knative (acting like a cloud providers in Vm world) .  Does it need the kubernetes worker nodes to be always running ? In this case, how does it  help with  cost saving? Please let me know if am missing something here.
 
 Thanks in advance

**Comment by dprotaso** [comment]
>  believe pods are created when there is a request and serve the traffic and scale down to zero - this is handled by knative (acting like a cloud providers in Vm world) 

Yup exactly. This is the goal of the Knative Serving subproject - https://knative.dev/docs/serving/

> Does it need the kubernetes worker nodes to be always running ? In this case, how does it help with cost saving?

Yes - Knative Serving has a control plane components that run as pods. You'll gain benefits when you size these components depending on the workloads that are running on the clusters. Also you can have your cluster autoscaler k8s nodes on demand
